src/metabase/api/table.clj

```diff
@@ -15,6 +15,7 @@
             [metabase.related :as related]
             [metabase.sync :as sync]
             [metabase.sync.concurrent :as sync.concurrent]
+            #_:clj-kondo/ignore
             [metabase.sync.field-values :as sync.field-values]
```

Comment:
default alias of `metabase.sync.field-values` is `field-values` which is also the default of the model for field values. Just ignore it instead of doing FQN.